### DISCUSSION TITLE
Lane Assistant

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#626] [Q&A] [answered]
I have no idea whether the OSM maps allow this... but a lane assistant would be a useful thing.

**Comment by devemux86** [comment, marked as answer]
@Leeb64 thanks for the suggestion.

We already discussed this in:
- https://github.com/devemux86/cruiser/discussions/420

See the answer in:
- https://github.com/devemux86/cruiser/discussions/110
